source/INI/PeanutButter.INI/INIFile.cs

```diff
@@ -28,6 +28,11 @@ public interface IINIFile
         /// </summary>
         string SectionSeparator { get; set; }
 
+        /// <summary>
+        /// Toggle whether an empty line is added at the bottom when persisted
+        /// </summary>
+        public bool AddEmptyLineAtTheBottom { get; set; }

```

Comment:
I think I'd prefer the name `AppendTrailingEmptyLine`. Your thoughts?

Reply:
English is not my native language, so let's go with your suggestion.👍